tools/pwgen.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+/**
+ * @author Nekohime <https://github.com/Nekohime>
+ * Password-generating utility
+*/
+
+import crypto from 'crypto';
+import * as db from '../common/db/utils.js';
+
+if (process.argv[2] !== undefined && process.argv[2].length >= 1) {
+  const password = process.argv[2];
+  const salt = crypto.randomBytes(128).toString('base64');
```

Comment:
`db.saltLength` instead of `128` (this way if it ever changes we only update it in one place)